File: Zestawy Zadań/C++/4 - Tablice/Zestaw 2 zadania/4.2.5 - Tablica z wyrazami, jesli E lub e.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    string tab[10];
    for (int i = 0; i < 10; i++)
    {
        cin >> tab[i];
    }
    for (int i = 0; i < 10; i++)
    {
        for (int j = 0; j <= tab[i].length(); j++)
        {
            if (tab[i][j] == 'E')
            {
                cout << tab[i] << '*' << endl;
                break;
            }
            else
            {
                if (tab[i][j] == 'e')
                {
                    cout << tab[i] << endl;
                    break;
                }
            }
        }
    }
    return 0;
        
}

File: Zestawy Zadań/Python/4 - Tablice/Zestaw 2 zadania/4.2.5 - Tablica z wyrazami, jesli E lub e.py
tab = [' '] * 10
for i in range(10):
    tab[i] = str(input("Podaj wyraz: "))

for i in range(10):
    if 'E' in tab[i]:
        print(tab[i], '*')
    elif 'e' in tab[i]:
        print(tab[i])
